Source/Core/DolphinWX/Debugger/DSPRegisterView.h

```diff
@@ -44,4 +44,7 @@ class DSPRegisterView : public wxGrid
 public:
 	DSPRegisterView(wxWindow* parent, wxWindowID id = wxID_ANY);
 	void Update() override;
+
+private:
+	CDSPRegTable* m_register_table;
```

Comment:
It might be worth a comment noting the ownership of m_register_table?  Not sure what the standard conventions are for wxWidgets.